tests/components/lutron_caseta/test_light.py

```diff
@@ -24,3 +29,41 @@ async def test_light_unique_id(
 
     state = hass.states.get(ra3_entity_id)
     assert state.state == STATE_ON
+
+
+async def test_previous_brightness(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test brightness tracked and restored."""
+    await async_setup_integration(hass, MockBridge)
+
+    # Initially, off with no brightness
+    caseta_entity_id = "light.kitchen_other_lights"
+
+    state = hass.states.get(caseta_entity_id)
+    assert state.state == STATE_OFF
+    assert state.attributes[ATTR_BRIGHTNESS] is None
+
+    # Turn on, expect this defaults to 50% (255/2) or 100% (255)
+    await hass.services.async_call(
+        LIGHT_DOMAIN,
+        SERVICE_TURN_ON,
+        {ATTR_ENTITY_ID: caseta_entity_id},
+        blocking=True,
+    )
+    await hass.async_block_till_done()
+
+    state = hass.states.get(caseta_entity_id)
+
+    assert state.state == STATE_ON
```

Comment:
I don't quite understand why this isn't changed to ON; if the service call above blocked until it was complete.